src/components/MultiHopTrade/components/SlippagePopover.tsx

```diff
@@ -59,26 +76,32 @@ export const SlippagePopover = () => {
         debounceFnc(value)
         setIsInvalid(false)
       }
-      setValue(value)
+      dispatch(swappers.actions.setSlippagePreferencePercentage(value))
```

Comment:
Since this is a user-selected one, stores it with the percentage vernacular to avoid going back and forth from/to decimal/percentage